Library/Homebrew/formula_lock.rb

```diff
@@ -35,6 +36,7 @@ def with_lock
 
   def get_or_create_lockfile
     if @lockfile.nil? || @lockfile.closed?
+      FileUtils.mkdir_p(LOCKDIR) unless File.directory?(LOCKDIR)
```

Comment:
`mkdir_p` already checks if the directory exists so we don’t need to test it ourselves.